src/sysupdate/sysupdated.c

```diff
@@ -335,7 +337,7 @@ static int job_on_exit(sd_event_source *s, const siginfo_t *si, void *userdata)
         else {
                 r = job_parse_child_output(TAKE_FD(j->stdout_fd), &json);
                 if (r < 0)
-                        sd_bus_error_set_errnof(&error, r, "Failed to parse JSON: %m");
+                        sd_bus_error_set_errnof(&error, r, "Failed to parse job worker output: %m");
```

Comment:
missing job id here?

Reply:
That's intentional.

The dbus client that this error delivered to doesn't need a job ID. Actually it'll probably have a better way to log about failure. For instance, updatectl will include the name of the target that failed which is a lot more actionable to the user on the CLI than a job ID.

And on the journal, where the job ID is useful, it'll be included when this error is printed out (a couple of lines below: `Job <id> failed with bus error, ignoring callback: <bus error message>`)